### PR TITLE
 feat: Loader

### PR DESCRIPTION
**Pull request by PeeyushPrashant**
Screenshot (135)

### CODE REVIEW

File: src/contexts/Product-context.js [hunk 1]
```diff
@@ -16,8 +16,10 @@ const ProductDataProvider = ({ children }) => {
   const [productState, dispatch] = useReducer(setProduct, {
     initialProduct: initialData,
   });
+  const [loader, setLoader] = useState(false);
 
   useEffect(() => {
+    let id;
```
Comment (Parul1999) [inline src/contexts/Product-context.js]: id is not used anywhere , it can be removed.

File: src/App.css [hunk 2]
```diff
@@ -7,6 +7,7 @@
   --red: #cf000f;
   --black: black;
```
Comment (anuj-kumary) [inline src/App.css:8]: Use hex code value

**Review by anuj-kumary — APPROVED** [review]
Code looks good and everything is well structured.

**Review by Parul1999 — COMMENTED** [review]
-- unused variables can be removed
-- loader looks good
-- Good short and precise components

**Pull request merged**